packages/web-components/src/components/va-minimal-header/va-minimal-header.scss

```diff
@@ -0,0 +1,35 @@
+@forward 'settings';
+
+.va-flex{
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 62.5em;
+  background-color: #182c54;
+  color:var(--color-white);
+  align-items: center;
+  padding: 20px 16px;
+}
+
+.va-logo {
+  margin-right: 16px;
+  background: url('../../assets/VA-seal.png') no-repeat;
+  min-height: 30px;
+  min-width: 48px;
+}
+
+.header-container {
+  padding-left: 1em;
+  border-left: 1px solid var(--color-white);
+
+  h1 {
+    font-size: 20px;
+    margin: 0;
+    font-weight: 700;
+  }
+  h2 {
+    font-size: 20px;
+    margin: 0;
+    font-weight: lighter;
```

Comment:
I think most of our component styles use numerical values for `font-weight`. It's been a minute since I've seen `lighter` used, so I had to read the [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) on it. Interesting that its computed value is derived from whatever font-weight is inherited. 

Are we able to use numerical values here? Or, if `lighter` is needed in order to get parity with the mockups, can you leave a comment for why its used on what its computed value ends up being?

Reply:
Updated to numerical.